scripts/sign.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+
+PROGRAM_NAME="${0##*/}"
+
+# Read input args
+while [[ "$#" -gt 0 ]]; do
+    case $1 in
+        -h|--help) HELP=true ;;
+        -d|--directory) content_dir="$2"; shift ;;
+        -s|--secret-key) secret_key_file=$2; shift ;;
+        -p|--public-key) public_key_file=$2; shift ;;
+        *) echo "Unknown parameter passed: $1"; exit 1 ;;
+    esac
+    shift
+done
```

Comment:
Do we want to _check for all required parameters_ here and send the HELP if we are missing anything? Or, alternatively, would we want to set some reasonable defaults for the `content_dir`, `secret_key_file`, and `public_key_file` and make some assumptions?

Reply:
I added defaults for `secret_key_file`, and `public_key_file`